ecopresto.php

```diff
@@ -119,7 +154,7 @@ private function controleLicence($id_eco, $typ)
 		else
 		{
 			if($id_eco != 'demo123456789demo123456789demo12')
-				Configuration::updateValue('ECOPRESTO_CONFIGURATION_OK', true);

```

Comment:
Can you please keep this configuration data ? ```ECOPRESTO_CONFIGURATION_OK``` allow us to know when a module is properly configured.